api/src/main/java/com/epam/pipeline/controller/pipeline/PipelineRunController.java

```diff
@@ -595,4 +595,18 @@ public Result<List<RunInfo>> loadRunsByParentId(@PathVariable(RUN_ID) final Long
     public Result<RunChartInfo> loadActiveRunsCharts(@RequestBody final RunChartFilterVO filter) {
         return Result.success(runApiService.loadActiveRunsCharts(filter));
     }
+
+    @PostMapping("/run/{runId}/network")
```

Comment:
lets rename it to `/run/{runId}/network/limit` to get instant sense what this method does